src/components/NumberTicker.tsx

```diff
@@ -0,0 +1,104 @@
+import * as React from 'react'
+import { Animated, StyleProp, StyleSheet, Text, TextStyle, View } from 'react-native'
+import { typeScale } from 'src/styles/fonts'
+
+interface CommonProps {
+  textHeight: number
+  textStyles?: StyleProp<TextStyle>
+}
+interface Props extends CommonProps {
+  finalValue: string
+  testID?: string
+}
+
+interface TickProps extends CommonProps {
+  startValue: number
+  endValue: number
+}
+
+interface TickTextProps extends CommonProps {
+  value: string
+}
+
+const numberRange = ['0', '1', '2', '3', '4', '5', '6', '7', '8', '9']
+
+function TickText({ value, textHeight, textStyles }: TickTextProps) {
+  return (
+    <View style={[styles.tickText, { height: textHeight }]}>
+      <Text style={[styles.text, textStyles]}>{value}</Text>
+    </View>
+  )
+}
+
+function Tick({ startValue, endValue, textHeight, textStyles }: TickProps) {
+  const animatedValue = new Animated.Value(startValue * textHeight * -1)
+  const transformStyle = { transform: [{ translateY: animatedValue }] }
+
+  Animated.timing(animatedValue, {
```

Comment:
what is the reason for not choosing reanimated?

Reply:
no reason, this solution was inspired by someone on youtube who chose Animated from RN. i think given the option, i'd choose RN over other libs anyway because RN will always exist but decreasing our dependencies is always a good thing?

Reply:
I agree that having fewer dependencies is good if we can achieve the same goals without them. but we are already using reanimated and I wonder if we should be consistent in our approach to animation.